cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -94,12 +96,8 @@ func extractExecutionState(
 			[]migrators.AccountBasedMigration{
 				// do account usage migration before and after as a sanity check.
 				&migrators.AccountUsageMigrator{},
-				cadenceDataValidation.PreMigration(),
-				migrators.NewAtreeRegisterMigrator(
-					rwf,
-				),
-				&migrators.DeduplicateContractNamesMigration{},
-				cadenceDataValidation.PostMigration(),
```

Comment:
Should these be removed?

Reply:
I think the plan is to have separate binaries: one for atree-migration and another for cadence value migrations. In that case, we don't need these.

Reply:
I see. That means the code will still be needed for that other binary? Where do we keep it? (just the merge of the SC feature branch merge will remove the code needed in that other binary)

Reply:
These two has to be kept in two branch AFAIU. Atree migration runs with pre-1.0 Cadence, where as cadence migrations run on 1.0. The source codes might also be not-compatible.